platform-sdk/docs/proposals/template.md

```diff
@@ -0,0 +1,119 @@
+# Summary
```

Comment:
Maybe everything should be one level higher and leave title 1 for the title of the proposal?

Reply:
I've added a title in markdown
```
Title Goes Heres
============
```

Reply:
@edward-swirldslabs I dont see this,
Either way, I think that adds the title as h1 too, sharing the same level as the others.

Reply:
`#` and `=====` are equivalent
`##` and `------` are equivalent

Reply:
I think this should still be addressed